src/java/org/apache/cassandra/tools/NodeTool.java

```diff
@@ -262,7 +262,8 @@ public int execute(String... args)
                .withCommand(CMSAdmin.InitializeCMS.class)
                .withCommand(CMSAdmin.ReconfigureCMS.class)
                .withCommand(CMSAdmin.Snapshot.class)
-               .withCommand(CMSAdmin.Unregister.class);
+               .withCommand(CMSAdmin.Unregister.class)
+               .withCommand(CMSAdmin.ResumeDropAccordTable.class);
```

Comment:
I couldn't decide whether this belonged as a subcommand  of `nodetool cms` or `nodetool accord`, wdyt?

Reply:
I think TCM is best.

Reply:
kk